espflash/src/lib.rs

```diff
@@ -14,7 +14,7 @@
 //! [espflash] can also be used as a library:
 //!
 //! ```toml
-//! espflash = { version = "2.1", default-features = false }
+//! espflash = { version = "4.0.0-dev", default-features = false }
 //! ```
```

Comment:
We could suggest using `cargo add espflash --no-default-features` to avoid having to bump the version or have the version outdated

Reply:
Probably yeah, I'm planning on rewriting the module documentation anyway so can be addressed in the future.